src/utils/datastructs.jl

```diff
@@ -26,43 +26,43 @@ end
 """
 struct ClustData <: TSData
  region::String
- years::Array{Int64}
+ years::Array{Int}
  K::Int
  T::Int
  data::Dict{String,Array}
- weights::Array{Float64}
+ weights::Array{AbstractFloat,2}
```

Comment:
```suggestion
 weights::Array{AbstractFloat}
```
Relevant for edge case with cluster number 1